MANUAL.md

```diff
@@ -1426,15 +1507,15 @@ The companion function `hasOption` returns true if an option is present. The fun
 
 > [!TIP]
 >
-> Custom option can be read from generated code, or from the schema passed to a plugin. See the example in
+> Custom options can be read from generated code, or from the schema passed to a plugin. See the example in
 > our [guide for writing plugins](#writing-plugins).
-
-> [!TIP]
 >
 > To learn more about custom options in Protobuf, see the [language guide][protobuf.dev/customoptions].
 
 ### Reflection API
 
+TODO: Can the API be used without the BSR? We should note requirements and link to the BSR docs if not.
```

Comment:
This is independent of the BSR. 

With the features listed here, it's possible to take full advantage of the BSR reflection API. It would be possible to write an SDK like [prototransform for Go](https://github.com/bufbuild/prototransform), or a command like `buf curl`. But doing that here in an example is a heavy lift.

I suggest we add an introductory paragraph right at the top `## Reflection` section:

> Reflection allows to inspect a schema and to dynamically manipulate data. In the following sections, we'll explain the core primitives, and write a function to redact sensitive information from messages.

Additionally, we can link to the BSR API, for example at the bottom of the `### Descriptors` section:
 
> [!TIP]
> 
> You can also fetch descriptors [from the Buf Schema Registry](https://buf.build/docs/bsr/reflection/overview).